activerecord/lib/active_record/errors.rb

```diff
@@ -178,10 +178,9 @@ class Rollback < ActiveRecordError
   class DangerousAttributeError < ActiveRecordError
   end
 
-  UnknownAttributeError = ActiveSupport::Deprecation::DeprecatedConstantProxy.new( # :nodoc:
-    'ActiveRecord::UnknownAttributeError', 
-    'ActiveModel::AttributeAssignment::UnknownAttributeError'
-  )
+  # Raised when unknown attributes are supplied via mass assignment.
+  class UnknownAttributeError < ActiveModel::UnknownAttributeError
```

Comment:
Rather than subclassing, let's just alias. There's no reason to have two separate exception classes.

```
UnknownAttributeError = ActiveModel::UnknownAttributeError
end
```